tensorrt_llm/_torch/models/modeling_dflash.py

```
@@ -0,0 +1,1295 @@
+            norms.append(norm)
+        self.fc_norm = nn.ModuleList(norms)
+        super().load_weights(weights, weight_mapper=weight_mapper, **kwargs)
+
+    def project_target_hidden(self, hidden_states):
+        """Project captured target features to the draft width: apply the per-aux
+        ``fc_norm`` to each hidden chunk, then ``fc`` + ``hidden_norm``."""
+        hidden_states = hidden_states.to(self.fc.weight.dtype)
+        fc_norm = getattr(self, "fc_norm", None)
+        if fc_norm is not None:
+            chunks = hidden_states.chunk(len(fc_norm), dim=-1)
+            hidden_states = torch.cat([norm(chunk) for norm, chunk in zip(fc_norm, chunks)], dim=-1)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)